psydac/linalg/tests/test_stencil_matrix.py

```diff
@@ -2001,193 +2001,6 @@ def test_stencil_matrix_2d_serial_backend_dot_3(dtype, n1, n2, p1, p2, s1, s2, P
     assert np.allclose(y1a, y1a_exact, rtol=1e-13, atol=1e-13)
     assert np.allclose(y2a, y2a_exact, rtol=1e-13, atol=1e-13)
 
-# ===============================================================================
```

Comment:
Why are these test cases deleted?

Reply:
Those tests try to do the transpose with different backend. But now the transpose function are stored in a kernel that should be pyccelised at the installation of psydac. So the `backend` passed to the `StencilMatrix`, will not change the transpose function. It is only defined during the initialization of the `StencilMatrix`. So those tests are useless